molgenis-data-import/src/main/java/org/molgenis/data/importer/ImportWriter.java

```diff
@@ -78,6 +78,9 @@ public ImportWriter(DataService dataService, PermissionSystemService permissionS
 	@Transactional
 	public EntityImportReport doImport(EmxImportJob job)
 	{
+		ImportedDataToBackendCompatibilityValidator validator = new ImportedDataToBackendCompatibilityValidator(dataService);
+		validator.validate(job.parsedMetaData.getEntities());
```

Comment:
Not entirely sure, but doesn't the validator need privileges to check ALL metadata for the presence of one of these entities. I.e. shouldn't this check be in the runAsSystem part?